pkg/collector/py/datadog_agent.c

```diff
@@ -117,7 +116,17 @@ static PyMethodDef datadogAgentMethods[] = {
  */
 static PyMethodDef utilMethods[] = {
   {"headers", (PyCFunction)Headers, METH_VARARGS, "Get basic HTTP headers with the right UserAgent."},
-  {"get_subprocess_output", (PyCFunction)get_subprocess_output, METH_VARARGS, "Run subprocess and return its output."},
+  {NULL, NULL}
+};
+
+/*
+ * _Util package is a private module for utility bindings
+ */
+static PyMethodDef _utilMethods[] = {
+  {"get_subprocess_output", (PyCFunction)get_subprocess_output, 
+      METH_VARARGS, "Run subprocess and return its output. "
+                    "This is a private method and should not be called directly. " 
+                    "Please use the utils.get_subprocess_output wrapper."},
```

Comment:
nit: technically the full path to the wrapper is `utils.subprocess_output.get_subprocess_output` :)